spk/shairport-sync/src/installer.sh

```diff
@@ -59,6 +61,11 @@ preupgrade ()
 {
     # Stop the package
     ${SSS} stop > /dev/null
+    
+    # Remove firewall config
```

Comment:
Not sure how those are executed, but it was meant for allowing to change ports within upgrades...?